docs/plugins/kodinfo.rst

```diff
@@ -0,0 +1,39 @@
+KodiNfo Plugin
+=================
+
+The plugin lets you create nfo files for Kodi music
+library and update it whenever you import an album into the library. 
```

Comment:
Is there some sort of link we can use here to explain more about what *.nfo files are and why you would want them?